app/services/fixity.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module Fixity
```

Comment:
So the local fixity work happens only in jobs, not services. But it still needs like it would be nice to differentiate between local and cloud stuff in the filename.

Same with the job; maybe `HandleCloudFixityJob` or something?

Reply:
I called it `UpdateFixityJob` assuming we'd spawn the same job for regular fixity checks down the line.

Reply:
We should make sure this intention is reflected in existing or new tickets